sdk/devcenter/azure-developer-devcenter/azure/developer/devcenter/_operations/_operations.py

```diff
@@ -3731,38 +3872,33 @@ def get_dev_box_action(
         if _stream:
             deserialized = response.iter_bytes()
         else:
-            if response.content:
-                deserialized = response.json()
-            else:
-                deserialized = None
+            deserialized = _deserialize(_models.DevBoxAction, response.json())
 
         if cls:
-            return cls(pipeline_response, cast(JSON, deserialized), {})  # type: ignore
+            return cls(pipeline_response, deserialized, {})  # type: ignore
 
-        return cast(JSON, deserialized)  # type: ignore
+        return deserialized  # type: ignore
 
     @distributed_trace
     def skip_dev_box_action(  # pylint: disable=inconsistent-return-statements
         self, project_name: str, user_id: str, dev_box_name: str, action_name: str, **kwargs: Any
     ) -> None:
```

Comment:
This operation didnt have a return type?

Reply:
No,it does not. You can see in the record the response body is null https://github.com/Azure/azure-sdk-assets/blob/95190a8afc4f186def04729c747b3a129d96e76c/net/sdk/devcenter/Azure.Developer.DevCenter/tests/SessionRecords/DevBoxesClientTests/SkipActionAndDeleteDevBoxSucceeds.json#L69C1-L96C7